src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -225,7 +225,9 @@ struct IsSameAppId : public IsSameApp {
     if (AreBothRemoteControl(other) && !IsSameDeviceType(other)) {
       return false;
     }
-    return strcasecmp(app_id_.c_str(), other->policy_app_id().c_str()) == 0;
+    return strncmp(app_id_.c_str(),
```

Comment:
@tkireva If we are looking for simple lexicographical compare, than std::basic_string::operator== does it, but looks much better in terms of C++, right?